src/molecules/form-fields/checkbox-form-field.test.tsx

```diff
@@ -20,4 +20,44 @@ describe("CheckboxFormField", () => {
         // Assert
         expect(getByLabelText(expected)).not.toBeNull();
     });
+
+    test("when errorsMessage prop set, renders with error message", () => {
+        // Arrange
+        const expected = faker.random.words();
+        const testErrorMessage = "testErrorMessage";
+
+        // Act
+        const { getByText } = render(
+            <CheckboxFormField
+                checked={false}
+                label={expected}
+                onChange={() => {}}
+                errorMessage={testErrorMessage}
+            />
+        );
+
+        // Assert
+        expect(getByText(testErrorMessage)).not.toBeNil();
+    });
+
+    test("when onChange set, calls handler upon change", () => {
+        // Arrange
+        let isChecked = false;
+        const expected = faker.random.word();
```

Comment:
Same as above ☝️ while it's being used here, `expected` leads me to believe we're looking for this value to be set somewhere in the assert step.

Reply:
Thanks for the feedback. I have updated the PR if you would like to take another look.